pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -665,6 +665,30 @@ def parse_args(args=None):
                               'https://developer.android.com/guide/'
                               'topics/manifest/'
                               'activity-element.html'))
+
+    ap.add_argument('--android-entrypoint', dest='android_entrypoint',
+                    default='org.kivy.android.PythonActivity',
+                    help='defines which java class sill be used for startup, usually a subclass of PythonActivity')
+    ap.add_argument('--android-apptheme', dest='android_apptheme',
+                    default='@android:style/Theme.NoTitleBar',
+                    help='defines which app theme should be selected for the main activity')
+    ap.add_argument('--add-java-source-compat', dest='java_source_compat',
```

Comment:
I don't like source compat and target compat options so much, see main comment.

Reply:
as suggested I introduced --add-compile-option